### DISCUSSION TITLE
Here's some extensive speed testing I did

### DISCUSSION THREAD

**Discussion by BBC-Esq** [bitsandbytes-foundation/bitsandbytes#1158] [General]
Here's some extensive testing that I did on some of the popular backends to run LLMs.  Please see an analogous post on the llama_cpp repo for the actual scripts.

![image](https://github.com/TimDettmers/bitsandbytes/assets/108230321/a63d44b4-ae3e-4122-83f3-b1529e313f9e)

Here's the link to the explanations:
https://github.com/ggerganov/llama.cpp/discussions/6373

**Comment by Titus-von-Koeller** [comment]
Really cool @BBC-Esq, thanks for sharing!
